src/Raven.Studio/typescript/viewmodels/database/tasks/importCollectionFromCsv.ts

```diff
@@ -44,6 +105,20 @@ class importCollectionFromCsv extends viewModelBase {
         this.importedFileName.extend({
             required: true
         });
+        
+        this.csvConfig().commentCharacter.extend({
+            required: {
+                onlyIf: (val: string) => this.csvConfig().allowComments() &&  _.trim(val) === ""
+            },
+            
+            validation: [
+                {
+                    validator: (val: string) => !this.csvConfig().allowComments() || (this.csvConfig().allowComments() && val && _.trim(val).length === 1),
+                    message: 'Please enter only one character for the comment character'
```

Comment:
The csv reader provides this option - I thought it would be nice to have

Reply:
ok

Reply:
I wouldn't use trim in validation, as if you put {space}{hash} it will pass validation and send to server 2 characters. Unless we trim this data before passing to csv import library ? 

Other thing, I assume we can't use // for comment, right? Only single char?

Reply:
1. right - I will add another trim before passing to server
2. can't use // - there is validation for entering only a single char

Reply:
where is this validation on our side or in library?

Reply:
Added validation on server side + another test